src/coreclr/vm/jitinterface.cpp

```diff
@@ -1920,15 +1920,15 @@ bool CEEInfo::canAllocateOnStack(CORINFO_CLASS_HANDLE clsHnd)
 unsigned CEEInfo::getClassAlignmentRequirement(CORINFO_CLASS_HANDLE type, bool fDoubleAlignHint)
 {
     CONTRACTL {
-        NOTHROW;
-        GC_NOTRIGGER;
+        THROWS; // Due to GetNativeLayoutInfo() call in getClassAlignmentRequirementStatic
```

Comment:
These comments should be unnecessary after the contract on `getClassAlignmentRequirementStatic` is fixed.